weave/Scripts/MenuControllers/SplashScreen.cs

```diff
@@ -0,0 +1,69 @@
+using Godot;
+using GodotSharper;
+using GodotSharper.AutoGetNode;
+using GodotSharper.Instancing;
+
+namespace Weave.MenuControllers;
+
+[Scene("res://Menus/SplashScreen.tscn")]
+public partial class SplashScreen : Node2D
```

Comment:
Moved the file, but it isn't recognizing it as the same file. Showing more changes than I did